portfolio/src/main/webapp/script.js

```diff
@@ -156,9 +156,46 @@ function updateButton() {  // eslint-disable-line no-unused-vars
 /* eslint-disable no-undef */
 
 function createMap() {
-  const map = new google.maps.Map(
-      document.getElementById('map'),
-      {center: {lat: 37.422, lng: -122.084}, zoom: 16});
+  const map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 37.740043, lng: -121.902838},
+    zoom: 13,
+  });
+
+  const onTheBorderMarker = new google.maps.Marker({
+    position: new google.maps.LatLng(37.703692, -121.885297),
+    title: 'On the Border',
+    map: map,
+  });
+  const bJsMarker = new google.maps.Marker({
+    position: new google.maps.LatLng(37.702720, -121.851593),
+    title: 'BJ\'s',
+    map: map,
+  });
```

Comment:
Add a function to help you create this Marker object, therefore you don't need to repeat the creation logic multiple times.

Reply:
Okay